Frends.AzureBlobStorage.ListBlobsInContainer/Frends.AzureBlobStorage.ListBlobsInContainer/ListBlobsInContainer.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.ComponentModel;
+using Azure.Storage.Blobs;
+using Azure;
+using Frends.AzureBlobStorage.ListBlobsInContainer.Definitions;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs.Models;
+using System.Collections.Generic;
+#pragma warning disable CS1573
+
+namespace Frends.AzureBlobStorage.ListBlobsInContainer
+{
+    public class AzureBlobStorage
+    {
+        /// <summary>
+        ///     List of blobs in container.
+        ///     [Documentation](https://tasks.frends.com/tasks#frends-tasks/Frends.AzureBlobStorage.ListBlobsInContainer)
+        /// </summary>
+        /// <param name="source">Source values.</param>
+        /// <returns>List of blobs in container with chosen listing structure.</returns>
+        /// <exception cref="Exception">Authentication exception.</exception>
+
+        public static async Task<Result> ListBlobsInContainer([PropertyTab] Source source)
+        {
+            BlobContainerClient blobContainerClient = null;
+            var blobsList = new List<Result>();
+            var authSas = false;
+
+            switch (source.AuthenticationMethod)
```

Comment:
I would say that this deserves an own method that just handles creating the client.